src/ORM/Locator/TableLocator.php

```diff
@@ -146,10 +146,7 @@ public function get($alias, array $options = [])
             $options += $this->_config[$alias];
         }
 
-        if (empty($options['className'])) {
-            $options['className'] = Inflector::camelize($alias);
-        }
-        $className = App::className($options['className'], 'Model/Table', 'Table');
+        $className = $this->_getClassName($alias, $options);
```

Comment:
Should be `$className = $options['className'] = $this->_getClassName($alias, $options);`